crates/mun_language_server/src/completion/expr.rs

```diff
@@ -57,9 +57,12 @@ mod tests {
             let foo_bar = 0;
             f$0
         }
-        "#,
-            Some(CompletionKind::Reference)
-        ));
+        "#
+        ), @r###"
```

Comment:
What does the `@` symbol mean?

Reply:
its a marker for insta so it knows where to place the inline snapshot.

Reply:
Thanks!